src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -202,8 +200,10 @@ function ReportActionItemMessageEdit(props) {
     const deleteDraft = useCallback(() => {
         debouncedSaveDraft.cancel();
         Report.saveReportActionDraft(props.reportID, props.action.reportActionID, '');
-        ComposerActions.setShouldShowComposeInput(true);
-        ReportActionComposeFocusManager.focus();
+        if (isFocusedRef.current) {
+            ComposerActions.setShouldShowComposeInput(true);
```

Comment:
QQ: Why haven't we utilized the `setShouldShowComposeInputKeyboardAware` in this case?

Reply:
Great catch. I think we should use `setShouldShowComposeInputKeyboardAware` in this case as well.

What do you think @s77rt ?

Reply:
I don't think this is a necessity. We should not delay showing the main composer unless we have to.

Reply:
ok yeah, just wanted to confirm we haven't missed this case, thanks.